src/main/java/org/mifos/processor/bulk/schema/SubscriptionOptionsDTO.java

```diff
@@ -0,0 +1,46 @@
+package org.mifos.processor.bulk.schema;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.*;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class SubscriptionOptionsDTO {
+
+    @JsonProperty("TTL")
+    private int tTL;
+
+    @JsonProperty("FIFO")
+    private boolean fIFO;
+
+    public static Builder builder = new Builder();
+
+    public static class Builder {
```

Comment:
Same comment as above. 
Additionally:  a class with just two instance variables usually need not call a builder unless these two are optional fields.
In this case, however, I accept the builder pattern as the builder is used for setting the default values for each instance variable.

Reply:
I added builder patter for the default value.

Reply:
updated the builder class name.